.github/workflows/tests.yml

```diff
@@ -30,13 +30,25 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-node@v2
 
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: nightly
+
+      - name: Cache .cargo
+        id: cargo-cache
+        uses: actions/cache@v2
+        with:
+          path: |
+            ./cargo/
```

Comment:
One last typo 😛 `~/.cargo/`

Reply:
Not sure if this is working 🤔

![image](https://user-images.githubusercontent.com/8050659/150599732-8e45d0d3-6973-45b8-b311-781e88d2398f.png)

Is `./.cargo` and `~/.cargo` same? Probably not, `./` is relative to repo.